fairscale/nn/misc/flatten_params_wrapper.py

```diff
@@ -137,9 +127,7 @@ def __getattr__(self, name: str) -> Any:
         except AttributeError:
             return getattr(self.module, name)  # fallback to wrapped module
 
-    def state_dict(
-        self, prefix: str = "", keep_vars: bool = False,
-    ) -> OrderedDict[str, Tensor]:
+    def state_dict(self, prefix: str = "", keep_vars: bool = False) -> "OrderedDict[str, Tensor]":  # type: ignore
```

Comment:
mypy overloading is supposed to be handled like that : declare the interfaces first, then the implementations ((see)[https://mypy.readthedocs.io/en/stable/more_types.html?highlight=overload#function-overloading]). For some reason I could not get it to agree with the stub, +OrderedDict have a different type depending on the python versions (only properly defined after 3.7.5 it seems). All in all I thought that # type: ignore was the most reasonable path